src/js/utils/background.js

```diff
@@ -76,7 +76,7 @@ export const backgroundStyle = (backgroundArg, theme) => {
         )
       ) || color;
       styles.push(css`
-        background-color: ${backgroundColor};
+        background: ${backgroundColor};
```

Comment:
This change breaks the fix for https://github.com/grommet/grommet/issues/2212.
I'm actually not so sure that we should be doing gradients as a `color`. Instead, that feels like something that should be handled via a `<Box background={...} />`.